keyboards/era/divine/info.json

```
@@ -0,0 +1,115 @@
+{
+    "manufacturer": "ERA",
+    "keyboard_name": "DIVINE 1.0.0",
+    "maintainer": "eerraa",
+    "bootloader": "rp2040",
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "build": {
+        "debounce_type": "sym_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "GP1",
+        "num_lock": "GP2",
+        "scroll_lock": "GP3",
+		"on_state": 0
```

Change tabs to spaces
```suggestion
        "on_state": 0
```